lib/MusicBrainz/Server/Entity/Language.pm

```diff
@@ -63,6 +63,7 @@ around TO_JSON => sub {
 
     my $json = $self->$orig;
     $json->{name} = $self->name;
+    $json->{iso_code_3} = $self->iso_code_3;
```

Comment:
I imagine it is because it's more consistent (BCP47 seems to use code 1 when available and if not code 3, which means a combination of 2 and 3 letter codes and seems to give unneeded priority to larger languages).

Reply:
About _consistency_, we provide BCP47 in JSON-LD property [inLanguage](https://schema.org/inLanguage).